.github/workflows/ci.yml

```
@@ -0,0 +1,174 @@
+name: OpenMRS ESM Admin Tools CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "17"
```

Node operates under a weird thing where even versions are LTS, odd versions are temporary so technically 17 is unsupported now. 18 is the current latest, but everything else is on 16, which is still supported for now.

```suggestion
          node-version: "16"
```